tools/test-js-with-node

```diff
@@ -66,6 +80,39 @@ except OSError:
     print('Bad command: %s' % (command,))
     raise
 
+NODE_COVERAGE_PATH = 'var/node-coverage/coverage.json'
+
+try:
+    with open(NODE_COVERAGE_PATH, 'r') as f:
+        coverage_json = ujson.load(f)
+        print("\nChecking enforced fully covered files...")
+        if not options.coverage:
+            print("Node coverage statistics may not be fully updated.")
+            print("Run `test-js-with-node --coverage` to update coverage statistics.\n")
+        for path in enforce_fully_covered:
+            if not (path in coverage_json): continue
+            line_coverage = coverage_json[path]['s']
+            missing_lines = []
+            for line in line_coverage:
+                if line_coverage[line] == 0:
```

Comment:
Oh, one more thing -- we should only do this coverage test if they're running the whole suite, not if they're doing just a single file.